.github/workflows/ci.yaml

```diff
@@ -49,6 +49,8 @@ jobs:
         node-version: ${{ matrix.node_version }}
     - name: Install dependencies
       run: yarn
+    - name: Disable hard-source-webpack-plugin
+      run: echo "::set-env name=HARP_NO_HARD_SOURCE_CACHE::true"
```

Comment:
Wow, that's a really weird syntax. I saw this action in the marketplace but might be a bit overkill:
https://github.com/marketplace/actions/set-env

Reply:
The proper syntax for _global_ env variables - like this one is to use `env` section which is above in this file:

```
env:
  HARP_NO_HARD_SOURCE_CACHE: true
```

Reply:
@zbigg  but that is if you always want to have it. set-env as step can be configured, so we could re-enable in certain cases.

Reply:
@zbigg  also `env` is not what we want in our case because as far as I know it doesn't propagate to the next steps

from https://docs.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#about-environment-variables

You can also use the set-env workflow command to set an environment variable that the following steps in a workflow can use.

Reply:
Well, in this case we run few `webpack` throughout flow and i think that _global_ and _constant_ `HARP_NO_HARD_SOURCE_CACHE` would do the trick with no harm to steps that ignore it.

The `::set-env` is useful if you need to derive env variable value from output of command in very specific point of pipeline. In this case, i don't see any reason why `yarn` or cache setup would suffer from having our variable set.

YMMV though :)